tabpy-server/server_tests/test_pwd_file.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import unittest
+from argparse import Namespace
+from tempfile import NamedTemporaryFile
+
+from tabpy_server.app.app import TabPyApp
+from tabpy_server.app.ConfigParameters import ConfigParameters
+
+try:
+    from unittest.mock import patch, call
+except ImportError:
+    from mock import patch, call
+
+
+class TestPasswordFile(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(TestPasswordFile, self).__init__(*args, **kwargs)
+        self.config_file = None
+        self.pwd_file = None
+        self.cwd = os.getcwd()
+        self.tabpy_cwd = os.path.join(self.cwd, 'tabpy-server', 'tabpy_server')

```

Comment:
With minimum supported Python version I strongly advise using pathlib ops instead as they really simplify calls like this. This entire snippet could be emulated as `pathlib.Path.cwd() / 'tabby-server/tabpy_server'`

Reply:
fixing this as well